client/my-sites/plugins/plugin.jsx

```diff
@@ -259,7 +259,7 @@ export default React.createClass( {
 				<div className="plugin__page">
 					{ this.displayHeader() }
 					<PluginMeta
-						notices={ [] }
+						notices={ {} }
```

Comment:
I think it is ok to pass an empty object here Since it is required by the other components down the line. I think once we have the refux stores we woun't have to worry about this so much and can remove them.